src/test/java/uk/gov/companieshouse/api/accounts/controller/DebtorsControllerTest.java

```diff
@@ -119,4 +130,134 @@ void createDebtorsBindingErrors() {
         assertNotNull(responseEntity);
         assertEquals(HttpStatus.BAD_REQUEST, responseEntity.getStatusCode());
     }
+
+    @Test
+    @DisplayName("Get debtors - success")
+    void getDebtorsSuccess() throws DataException {
+
+        when(request.getAttribute(AttributeName.TRANSACTION.getValue())).thenReturn(transaction);
+        when(debtorsService.generateID(COMPANY_ACCOUNTS_ID)).thenReturn(DEBTORS_ID);
+
+        ResponseObject responseObject = new ResponseObject(ResponseStatus.FOUND,
+                debtors);
+        when(debtorsService.findById(DEBTORS_ID, request))
+                .thenReturn(responseObject);
+
+        ResponseEntity responseEntity = ResponseEntity.status(HttpStatus.FOUND)
+                .body(responseObject.getData());
+        when(apiResponseMapper.mapGetResponse(responseObject.getData(), request))
+                .thenReturn(responseEntity);
+
+        ResponseEntity returnedResponse =
+                controller.get(COMPANY_ACCOUNTS_ID, request);
+
+        assertNotNull(returnedResponse);
+        assertEquals(HttpStatus.FOUND, responseEntity.getStatusCode());
+        assertEquals(debtors, responseEntity.getBody());
+    }
+
+    @Test
+    @DisplayName("Get debtors - data exception thrown")
+    void getDebtorsDataException() throws DataException {
+
+        when(request.getAttribute(AttributeName.TRANSACTION.getValue())).thenReturn(transaction);
+        when(debtorsService.generateID(COMPANY_ACCOUNTS_ID)).thenReturn(DEBTORS_ID);
+
+        DataException dataException = new DataException("");
+        when(debtorsService.findById(DEBTORS_ID, request))
+                .thenThrow(dataException);
+
+        ResponseEntity responseEntity = ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).build();
+        when(apiResponseMapper.map(dataException)).thenReturn(responseEntity);
+
+        ResponseEntity returnedResponse = controller.get(COMPANY_ACCOUNTS_ID, request);
+
+        assertNotNull(returnedResponse);
+        assertEquals(HttpStatus.INTERNAL_SERVER_ERROR, responseEntity.getStatusCode());
+        assertNull(responseEntity.getBody());
+    }
+
+    @Test
+    @DisplayName("Update debtors - no small full link")
+    void updateDebtorsNoSmallFullLink() {
+
+        when(request.getAttribute(AttributeName.SMALLFULL.getValue())).thenReturn(smallFull);
+        when(smallFull.getLinks()).thenReturn(smallFullLinks);
+        when(smallFullLinks.get(SmallFullLinkType.DEBTORS_NOTE.getLink())).thenReturn(null);
+
+        ResponseEntity responseEntity =
+                controller.update(debtors, bindingResult, COMPANY_ACCOUNTS_ID, request);
```

Comment:
Can we please follow the starndards regarding spaces, here and every file?
https://github.com/companieshouse/styleguides/blob/master/java.md#indentation